cmd/evm/internal/t8ntool/execution.go

```diff
@@ -213,7 +213,7 @@ func (pre *Prestate) Apply(vmConfig vm.Config, chainConfig *params.ChainConfig,
 			rejectedTxs = append(rejectedTxs, &rejectedTx{i, errMsg})
 			continue
 		}
-		msg, err := core.TransactionToMessage(tx, signer, pre.Env.BaseFee, core.MessageCommitMode)
+		msg, err := core.TransactionToMessage(tx, signer, pre.Env.BaseFee, core.NewMessageCommitContext(nil)) // TODO: do we need to specify wasm targets here if chainConfig.IsArbitrum == false?
```

Comment:
do we need to specify wasm targets here if chainConfig.IsArbitrum == false?

Reply:
per other comments, I think we should allow init with nil to mean localTarget
Also, if stylus isn't active, it should't be a problem to really have nil there